unity-renderer/Assets/DCLPlugins/ECS7/InternalECSComponents/Interfaces/Extensions/InternalSceneBoundsCheckExtensions.cs

```diff
@@ -0,0 +1,158 @@
+        }
+
+        public static void SetAudioSource(this IInternalECSComponent<InternalSceneBoundsCheck> sbcInternalComponent,
+            IParcelScene scene, IDCLEntity entity, AudioSource audioSource)
+        {
+            var model = sbcInternalComponent.GetFor(scene, entity)?.model ?? new InternalSceneBoundsCheck();
+            model.audioSource = audioSource;
+
+            sbcInternalComponent.PutFor(scene, entity, model);
+        }
+
+        public static void RecalculateEntityMeshBounds(this IInternalECSComponent<InternalSceneBoundsCheck> sbcInternalComponent,
+            IParcelScene scene, IDCLEntity entity)
+        {
+            var model = sbcInternalComponent.GetFor(scene, entity)?.model;
+
+            if (model == null) return;
+
+            // Clean existing bounds object
+            model.entityLocalMeshBounds.size = Vector3.zero;
+
+            // Note: the center shouldn't be modified beyond this point as it affects the bounds relative values
+            model.entityLocalMeshBounds.center = entity.gameObject.transform.position;
+
+            // Encapsulate with global bounds
+            if (model.renderers != null)
+            {
+                for (var i = 0; i < model.renderers.Count; i++)
+                {
+                    model.entityLocalMeshBounds.Encapsulate(model.renderers[i].bounds);
+                }
+            }
+
+            if (model.physicsColliders != null)
+            {
+                for (var i = 0; i < model.physicsColliders.Count; i++)
+                {
+                    model.entityLocalMeshBounds.Encapsulate(GetColliderBounds(model.physicsColliders[i]));
+                }
+            }
+
+            if (model.pointerColliders != null)
+            {
+                for (var i = 0; i < model.pointerColliders.Count; i++)
+                {
+                    model.entityLocalMeshBounds.Encapsulate(GetColliderBounds(model.pointerColliders[i]));
+                }
+            }
+
+            // Turn min-max values to local/relative to be usable when the entity has moved
+            Vector3 entityPosition = entity.gameObject.transform.position;
+            model.entityLocalMeshBounds.SetMinMax(model.entityLocalMeshBounds.min - entityPosition,
+                model.entityLocalMeshBounds.max - entityPosition);
+
+            sbcInternalComponent.PutFor(scene, entity, model);
+        }
+
+        private static Bounds GetColliderBounds(Collider collider)
+        {
+            Bounds returnedBounds = collider.bounds;
+
+            // Disabled colliders return a size-0 bounds object, so we enable it only to get its bounds
+            if (!collider.enabled)
+            {
+                GameObject colliderGO = collider.gameObject;
+                int colliderLayer = colliderGO.layer;
+
+                // Temporarily change the collider GO layer to avoid it colliding with anything.
+                colliderGO.layer = PhysicsLayers.gizmosLayer;
```

Comment:
Is there any other way to achieve the same?
If there is not, then we could change the layer naming, since builder is not supported any more `gizmosLayer` is obsolete

Reply:
removed here https://github.com/decentraland/unity-renderer/commit/8ea6be3b596911023ae348131288949702258e44